src/axom/primal/geometry/Tetrahedron.hpp

```diff
@@ -45,11 +45,11 @@ class Tetrahedron
   { }
 
   /*!
-   * \brief Custom Constructor. Creates a tetrahedron from the 4 points A,B,C,D.
-   * \param [in] A point instance corresponding to vertex A of the tetrahedron.
-   * \param [in] B point instance corresponding to vertex B of the tetrahedron.
-   * \param [in] C point instance corresponding to vertex C of the tetrahedron.
-   * \param [in] D point instance corresponding to vertex D of the tetrahedron.
+   * \brief Tetrahedron constructor from 4 points A,B,C,D.
```

Comment:
Should there be a comment about the ordering of the vertices, or does it not matter?

Reply:
done -- thanks